transform/mattermost-analytics/models/marts/product/licenses/dim_latest_server_customer_info.sql

```diff
@@ -0,0 +1,18 @@
+select
+    server_id
+    , license_id
+    , installation_id
+    , coalesce(license_company_name, cloud_company_name) as company_name
+    , coalesce(license_contact_email, cloud_contact_email) as contact_email
+    , coalesce(license_sku, cloud_sku) as sku
+    , license_expire_at as license_expire_at
```

Comment:
nit: remove the aliasing here and below since we don't do it in other places (e.g. ast_license_telemetry_date or others).